client/js/tracking.js

```diff
@@ -0,0 +1,16 @@
+import { on, nodeList } from './util';
+import { trackEvent } from './utils/track-event';
+
+// Transporter
+on('body', 'click', '[data-track="transporter"]', (event) => {
+  const transporter = event.target.closest('[data-track="transporter"]');
+  const promos = nodeList(transporter.querySelectorAll('.promo'));
+  const promo = event.target.closest('.promo');
+  const link = promo ? promo.href : null;
+  const position = promo ? promos.indexOf(promo) : null;
+  const properties = {id: transporter.id, link, position};
+
+  if (link) {
+    trackEvent({name: 'Transporter', properties});
+  }
+});
```

Comment:
Thoughts for potentially another PR.

I wonder if there is something more generic we can do here, potentially adding this into the DOM:
```twig
<div data-component="Transformer" data-component-id="12">
  {% for item in items %}
  <a data-track-click='{ "position": loop.index }'>Things</a>
  {% .endfor %}
</div>
```

Then when someone clicks that - we can figure out the component and it's ID (something that will probably be generic)* and send over the data.

As mentioned this wields the same results, so no matter until we do it again.

* I wonder if this is a chance to fix my mistake and call it `componentId`? We can loose a bit of data around the image galleries.